tests/testthat/test-directionchange.R

```diff
@@ -0,0 +1,20 @@
+
+context("test direction change functions")
+
+# Note: the line below does not appear correctly in R Studio
+
+test_that("char_tortl works.", {
+    skip_on_os("windows")
+    txt <- "מדינת ישראל נוסדה בשנת 1948."
+    expect_equal(char_tortl(txt), "מדינת ישראל נוסדה בשנת 1948.\u200F")
+    
+    txt <- "!מדינת ישר computer וסדה בשנת 1948."
+    # expect_equal(char_tortl(txt), "!מדינת ישר computer וסדה בשנת 1948.")
```

Comment:
What's the expectation here, when there is a ltr word sequence sandwiched in between two sequences of rtl words?

Note that I could not even get this to edit properly on my console! and I am not sure what the correct expectation is.